### ISSUE TITLE
Error with lme4 models with offset and re.form = NULL

### ISSUE THREAD

**Issue by florianhartig** [florianhartig/DHARMa#335]
Reported by a user. Reproducible example:

```
library(DHARMa)
dat <- createData()

fit <- glmer(observedResponse ~ (1|group) + offset(Environment1), data = dat, family = "poisson")
summary(fit)
simulateResiduals(fit, plot = T, re.form = NULL)
```

**Comment by bbolker** [comment]
I think https://github.com/lme4/lme4/commit/8c8ab5c95c9438f41667052602b101f7b78a52de should fix this ... although (a) I'm not sure it's 100% bulletproof (although I couldn't break it as easily as I thought I'd be able to) and (b) can't give a guarantee at the moment about when this will make it to CRAN (although I have my arm twisted; last CRAN submission was 7 May 2022, so should probably wait a couple of weeks at least)

**Comment by florianhartig** [comment]
Great, thanks! I'll leave this open for a moment to check with the next CRAN version!

**Comment by florianhartig** [comment]
This was fixed in glmmTMB 1.1.4 -> closed!
